### PR TITLE
Warning about unknown property happened when creating element

### LINKED ISSUE #2887: Detect React DOM element with a wrong attribute name
Dear all,

I'm pretty new to React. For some reasons I want to use it without JSX, and I faced an issue I can't resolve: if in the react's element properties you have a `class` key instead of `className` key (for example), in the console you'll see a warning: `Warning: Unknown DOM property class. Did you mean className?`. Okay, I really mean `className`, now how can I find it? I mean where I supposed to look first? In the fb's react plugin for chrome I don't see "wrong" attribute, warning don't give me any stack trace either. If I have a huge DOM it's very hard to go thru it manually, without any hints. Does it possible to find any workaround?

[Here you can see an example](http://jsbin.com/zarimegenu/1/)

### PR DESCRIPTION
**Pull request by Sumei1009**
fixes #2887

### CODE REVIEW

File: src/renderers/dom/shared/ReactDOMDebugTool.js [hunk 1]
```diff
@@ -61,6 +59,4 @@ var ReactDOMDebugTool = {
   },
 };
 
-ReactDOMDebugTool.addDevtool(ReactDOMUnknownPropertyDevtool);
```
Comment (jimfb) [inline src/renderers/dom/shared/ReactDOMDebugTool.js]: Why are you removing the devtool?  This appears to be going in the wrong/opposite direction.

File: src/isomorphic/classic/element/ReactElement.js [hunk 2]
```diff
@@ -146,6 +147,62 @@ ReactElement.createElement = function(type, config, children) {
...
+
+        var registrationName = (
+          EventPluginRegistry.possibleRegistrationNames.hasOwnProperty(
+            lowerCasedName
+          ) ?
+            EventPluginRegistry.possibleRegistrationNames[lowerCasedName] :
+            null
+        );
+
+        warning(
```
Comment (gaearon) [inline src/isomorphic/classic/element/ReactElement.js]: I think we want to use the “devtool” API for the new warnings. You can check how `ReactDOMUnknownPropertyDevtool` currently works for the context—it just needs to be called during the element creation.

**Comment by Sumei1009** [comment]
I modified some method so that ReactDOMUnknownPropertyDevtool would be called on element creation. Is it the right way?

**Comment by gaearon** [comment]
Can you please rebase so it merges cleanly, and fix the lint issues? You can run `npm run lint` locally and see why ESLint is failing. Thanks!

**Comment by jimfb** [comment]
Ping @Sumei1009

**Comment by kevinSuttle** [comment]
Just got bit by this `autof(F)ocus` thing. Would love to see this merged.

**Pull request closed without merging**